presto-hive-common/pom.xml

```diff
@@ -18,5 +18,91 @@
             <groupId>com.facebook.presto</groupId>
             <artifactId>presto-spi</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
many dependencies in this file won't be necessary; compilation will fail

Reply:
I did build and tested locally. It was fine.